actors/builtin/miner/miner_actor.go

```diff
@@ -1541,18 +1542,39 @@ func terminateSectors(rt Runtime, sectorNos *abi.BitField, terminationType power
 			rt.Abortf(exitcode.ErrIllegalState, "failed to expand faults")
 		}
 
+		var skippedSectorNos []uint64 // sectors we're not terminating because they either don't exist or have expired.
 		pledgeRequirementToRemove := abi.NewTokenAmount(0)
 		err = sectorNos.ForEach(func(sectorNo uint64) error {
 			sector, found, err := st.GetSector(store, abi.SectorNumber(sectorNo))
 			if err != nil {
 				return fmt.Errorf("failed to load sector %v: %w", sectorNo, err)
 			}
+
+			// If the sector wasn't found, skip termination. It may
+			// have already expired, have been terminated due to a
+			// faults, etc.
 			if !found {
-				rt.Abortf(exitcode.ErrNotFound, "no sector %v", sectorNo)
+				// XXX: Should we fail if the termination reason
+				// is "manual termination"? Or should we just
+				// let termination be idempotent?
```

Comment:
Let's just let it go. The manual declaration might have raced something else, potentially with a looong blockchain delay. We shouldn't deny any other terminations they're trying to achieve at the same time.